chainindex/ddls.go

```diff
@@ -0,0 +1,55 @@
+package chainindex
+
+const DefaultDbFilename = "chainindex.db"
+
+const (
```

Comment:
Looking through this list, I think it might be helpful if we adopt a naming convention that helps quickly grok what the stmt might be doing. `Insert` is the only consistent prefix here but we could do `Get` or `Select` for all of the `SELECT` statements, `Delete` or `Remove` for all of the `DELETE`s and `UPDATE` for the `UPDATE`s. The `SELECT` statements that are boolean could be `Has` if `Get` is awkward.
I've experienced very mild frustration across the existing ones when I have to trace back to the original transaction to even get a basic understanding of what it's doing.